src/components/Box.js

```diff
@@ -11,13 +11,19 @@ const Box = styled.section`
   background-color ${props => props.theme.backgroundColorRaised};
   border-radius: 6px;
   box-shadow: 0 8px 16px 0 rgba(0,0,0,0.05);
+  transition: all 0.05s ease-out;
 
   @media (min-width: ${baselineBreakpoint}px) {
     padding-top: ${4 * baselineLarge}px;
     padding-bottom: ${5 * baselineLarge}px;
-    padding-left: ${props => props.theme.contentPaddingLarge}px;
-    padding-right: ${props => props.theme.contentPaddingLarge}px;
+    padding-left: ${5 * baselineLarge}px;
+    padding-right: ${5 * baselineLarge}px;
```

Comment:
Again, baseline variables shouldn't be used for horizontal spacing. If the content padding theme values are not good, you can just inline the values here.